editor/editor_build_profile.cpp

```diff
@@ -194,11 +214,15 @@ String EditorBuildProfile::get_build_option_description(BuildOption p_build_opti
 		TTRC("3D Nodes as well as RenderingServer access to 3D features."),
 		TTRC("2D Physics nodes and PhysicsServer2D."),
 		TTRC("3D Physics nodes and PhysicsServer3D."),
-		TTRC("Navigation, both 2D and 3D."),
+		TTRC("2D Navigation nodes and NavigationServer2D."),
+		TTRC("3D Navigation nodes and NavigationServer3D."),
 		TTRC("XR (AR and VR)."),
-		TTRC("RenderingDevice based rendering (if disabled, the OpenGL back-end is required)."),
-		TTRC("OpenGL back-end (if disabled, the RenderingDevice back-end is required)."),
-		TTRC("Vulkan back-end of RenderingDevice."),
+		TTRC("RenderingDevice based rendering (if disabled, the OpenGL backend is required)."),
+		TTRC("OpenGL backend (if disabled, the RenderingDevice backend is required)."),
+		TTRC("Vulkan backend of RenderingDevice."),
+		TTRC("Direct3D 12 backend of RenderingDevice (Windows)."),
+		TTRC("Metal backend of RenderingDevice (macOS and iOS)."),
+		TTRC("Advanced GUI nodes: Dialogs, Tree, RichTextLabel, TextEdit, GraphEdit, and more."),
```

Comment:
We'll need better documentation on this somewhere, currently you need to read the source code to know what this impacts. It also seems pretty arbitrary and removing stuff like RichTextLabel likely makes it unusable for most games, even if they aren't UI heavy.